### ISSUE TITLE
A0399 specimen and disease name

### ISSUE THREAD

**Issue by kmartinez834** [biomarker-ontology/biomarkerontology#3]
https://data.tst.oncomx.org/allbiomarkers/biomarker/A0399

Urinary bladder cancer panel lists specimen type and disease name as Liver Cancer, please confirm if this is correct.

![image](https://user-images.githubusercontent.com/90285371/214386818-20d69107-bdbe-4948-8929-7380946e687e.png)

**Comment by danlymangw** [comment]
Yes, this is an error. 
I saw this error in the file before sending it to you and thought I had fixed it; good catch.

Corrections:
Specimen type:	urine (UN:0001088)
Disease name:		urinary bladder cancer (DOID:11054)

**Comment by kmartinez834** [comment]
Please take a look at these panel files and let me know if we should revert back to the "original" from April 2022:

Current file: https://data.tst.oncomx.org/OMX_000095

April 2022 file: 
[human_cancer_biomarker_panel_vysis_urinary_bladder_cancer_042522.csv](https://github.com/biomarker-ontology/biomarkerontology/files/10510782/human_cancer_biomarker_panel_vysis_urinary_bladder_cancer_042522.csv)

**Comment by danlymangw** [comment]
Yes, all these values (Specimen type: lung (UN:0002048), Disease name: lung
cancer (DOID:1324)) are incorrect.

Correct values:
Specimen type: urine (UN:0001088)
Disease name: urinary bladder cancer (DOID:11054)

Also, I've noticed some issues with BCOs that we should discuss

Daniel Lyman, PhD

Senior Research Associate

Cancer Biomarkers Project Lead, HIVE-Lab

MS in Bioinformatics and Biochemistry Program Coordinator

Department of Biochemistry and Molecular Medicine

School of Medicine & Health Sciences

The George Washington University

Ross Hall

2300 Eye Street N.W.

Washington, DC 20037

Phone lab: 202-994-3639

MS Program Office hours: T/W 9:00am-5:00pm
